HW_6/HW_7.py

```diff
@@ -0,0 +1,175 @@
+
+import datetime
+import random
+import datefinder
+import HW_3_updated as normalization
+import sys, os
+import csv
+import re
+from collections import Counter
+
+sys.path.append('C:\\Users\\Kateryna_Kulmatytska\\PycharmProjects\\Python_HW\\HW_6')
+
+class Article:
+    def __init__(self, header, text):
+        self.header = header
+        self.text = text
+
+    def print_to_file(self, formatted_text, file_name):
+        with open(file_name, "a") as file:
+            file.write(formatted_text)
+            with open(file_name, "r") as file_updated:
+                content = file_updated.read()
+                words = re.findall(r"[\w']+", content.lower())
+                d = dict()
+                for word in words:
+                    if word.isalpha():
+                        if word in d:
+                            d[word] = d[word] + 1
+                        else:
+                            d[word] = 1
+                with open('words_count.csv', "w", newline='') as words_count_file:
+                    for key, value in d.items():
+                        writer = csv.writer(words_count_file, delimiter='-')
+                        writer.writerow([f'{key}', f'{value}'])
+
+                with open('letters_count.csv', "w", newline='') as letters_count_file:
+                    headers = ['Letter', 'Count_all', 'Count_uppercase', 'Percentage']
+                    writer = csv.DictWriter(letters_count_file,  fieldnames=headers)
+                    writer.writeheader()
+                    letters_lowercase = Counter(letter for line in content
+                                      for letter in line.lower()
+                                      if letter.isalpha())
+                    letters_general = Counter(letter for line in content
+                                      for letter in line
+                                      if letter.isalpha())
+                    letters_total = len(re.findall("[a-zA-Z]", content))
+                    for letter, values in letters_lowercase.items():
+                        writer.writerow({f'Letter': letter, f'Count_all': letters_lowercase.get(letter), f'Count_uppercase': letters_general.get(letter.upper()),
```

Comment:
Also it'd be great to replace empty values by "0" if there is no uppercase symbols